website/src/components/home/highlights.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import Highlights from './highlights'
+import renderer from 'react-test-renderer'
+
+describe('Higlights', () => {
+  it('renders the right HTML', () => {
+    const tree = renderer.create(<Highlights />).toJSON()
```

Comment:
it's the first time I see a unit test on HTML 🎉  (not that I've seen much code in this area)